src/bootstrap/builder.rs

```diff
@@ -725,7 +725,11 @@ impl<'a> Builder<'a> {
             .env("CFG_RELEASE_CHANNEL", &self.config.channel)
             .env("RUSTDOC_REAL", self.rustdoc(compiler))
             .env("RUSTDOC_CRATE_VERSION", self.rust_version())
-            .env("RUSTC_BOOTSTRAP", "1");
+            .env("RUSTC_BOOTSTRAP", "1")
+            .arg("-Dinvalid_codeblock_attributes");
```

Comment:
This shouldn't be `-D...` but rather `-W`, with `-Dwarnings` upgrading that only when `--deny=warnings` or equivalent is passed to x.py.

Reply:
Same answer as I provided [here](https://github.com/rust-lang/rust/pull/71670#discussion_r455874412).